client/src/components/ProtectedRoute.js

```diff
@@ -37,7 +37,7 @@ function ProtectedRoute({ children, ...rest }) {
         ) : (
           <Redirect
             to={{
-              pathname: "/signin",
+              pathname: "signin",
```

Comment:
please check if this is the correct format for pathname without '/'

Reply:
Yes